src/issue-filing/common/create-issue-details-builder.ts

```diff
@@ -10,32 +11,48 @@ export const createIssueDetailsBuilder = (markup: MarkupFormatter): IssueDetails
     const getter = (environmentInfo: EnvironmentInfo, data: CreateIssueDetailsTextData): string => {
         const result = data.ruleResult;
 
-        const text = [
-            markup.sectionHeader('Issue'),
-            `${markup.snippet(result.help)} (${markup.link(result.helpUrl, result.ruleId)})`,
+        const { howToFixSection, link, sectionHeader, snippet, sectionHeaderSeparator, footerSeparator, newLine } = markup;
 
-            markup.sectionHeader('Target application'),
-            markup.link(data.pageUrl, data.pageTitle),
+        const lines = [
+            sectionHeader('Issue'),
+            sectionHeaderSeparator(),
+            `${snippet(result.help)} (${link(result.helpUrl, result.ruleId)})`,
+            newLine(),
```

Comment:
`newLine` seems a little plaintext-specific; maybe `sectionSeparator` would be more consistent with the other similar concepts?

Reply:
`sectionSeparator` makes more sense definitely. Updated.